### PR TITLE
Feature ver wye labs próximos 159956594

### PR DESCRIPTION
**Pull request by franbe7**
This branch added the index endpoint in wyelab controller, index route and tests

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,10 +2,12 @@
   resources :alerts, only: [:create]
   resources :groups, only: [:create, :show, :index]
   resources :users, only: [:create, :show, :update]
+  resources :wyelabs, only: [:create]
   post 'group/:group_id/users', to: 'user_groups#create', as: 'user_group'
 
   get 'users', to: 'users#index'
   get 'health', to: 'health#show'
+  get 'wyelabs', to: 'wyelabs#index'
```
Comment (Gimli76) [inline config/routes.rb]: You can put this route on the wyelabs resource, as follows:
`resources :wyelabs, only: [:create, :index]`

Comment (franbe7) [reply config/routes.rb]: Done

File: config/routes.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@
   resources :alerts, only: [:create]
   resources :groups, only: [:create, :show, :index]
   resources :users, only: [:create, :show, :update]
+  resources :wyelabs, only: [:create, :index]
```
Comment (joaquinvillar) [inline config/routes.rb]: create must not be here

File: spec/controllers/wyelab_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
+RSpec.describe WyelabsController, type: :controller do
+
+  describe "POST #create" do
```
Comment (joaquinvillar) [inline spec/controllers/wyelab_controller_spec.rb]: create must not be here

File: spec/controllers/wyelab_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+      end
+
+      it "creates a new Wyelab" do
+        allow(controller).to receive(:authorize)
+        post :create, params: no_title
+        expect(response).to have_http_status(:unprocessable_entity)
+        expect(response.content_type).to eq('application/json')
+      end
+    end
+  end
```
Comment (joaquinvillar) [inline spec/controllers/wyelab_controller_spec.rb]: missing index test

File: spec/models/wyelab_spec.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Wyelab, type: :model do
```
Comment (joaquinvillar) [inline spec/models/wyelab_spec.rb]: this test are for the create feature

File: spec/routing/wyelab_routing_spec.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe WyelabsController, type: :routing do
+  describe "routing" do
+    it "routes to #create" do
+      expect(post: "/wyelabs").to route_to("wyelabs#create")
```
Comment (joaquinvillar) [inline spec/routing/wyelab_routing_spec.rb]: this is from other feature

File: app/controllers/wyelabs_controller.rb [hunk 7]
```diff
@@ -0,0 +1,26 @@
+class WyelabsController < ApplicationController
+  # POST /wyelabs
+  def create
```
[the remark this answers is not in the document]
Comment (Gimli76) [reply app/controllers/wyelabs_controller.rb]: Most of the things that are from #25 should disappear once said feature is merged to development and this branch rebased.

File: app/controllers/wyelabs_controller.rb [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+        render json: @wyelab, status: :created
+      else
+        render json: @wyelab.errors, status: :unprocessable_entity
+      end
+    end
+  end
+
+  # GET /wyelabs
+  def index
+    @wyelabs = Wyelab.all.order(:created_at).reverse_order
```
[the remark this answers is not in the document]
Comment (franbe7) [reply app/controllers/wyelabs_controller.rb]: i'm agree

File: app/controllers/wyelabs_controller.rb (hunk 7 above)
[the remark this answers is not in the document]
Comment (franbe7) [reply app/controllers/wyelabs_controller.rb]: I had asked this and told me that it wasn't necessary. 
But i change it

File: app/controllers/wyelabs_controller.rb (hunk 8 above)
[the remark this answers is not in the document]
Comment (franbe7) [reply app/controllers/wyelabs_controller.rb]: I'm agree

**Pull request merged**